website/src/app/api.service.ts

```diff
@@ -1,4 +1,5 @@
 import {HttpClient, HttpErrorResponse, HttpParams} from "@angular/common/http";
+import { Icu } from "@angular/compiler/src/i18n/i18n_ast";
```

Comment:
I don't see where this import is getting used. If it's not used let's get rid of it